mopro-ios/MoproKit/Example/MoproKit/ViewController.swift

```diff
@@ -160,3 +171,28 @@ func bytesToBits(bytes: [UInt8]) -> [Int32] {
     }
     return bits
 }
+
+// TODO: should handle 254-bit input
+func serializeOutputs(_ int32Array: [Int32]) -> [UInt8] {
```

Comment:
Actually why do we have to do serializeOutputs here when we don't do this for inputs? Shouldn't it be the same?